frontend/packages/dev-console/src/components/topology/Topology.tsx

```diff
@@ -35,6 +39,69 @@ export default class Topology extends React.Component<TopologyProps, State> {
     });
   };
 
+  setNodeGroup = (dc: ResourceProps, labels: { [key: string]: string }) => {
+    const patch = [
+      {
+        op: 'replace',
+        path: '/metadata/labels',
+        value: labels,
+      },
+    ];
+
+    return k8sPatch(
+      dc.kind === 'DeploymentConfig' ? DeploymentConfigModel : DeploymentModel,
```

Comment:
Move this logic to a utility.
We need to update ALL the resources related to this node and change the `part-of` label. Not just the DC itself.